lib/ledger.js

```diff
@@ -67,11 +70,14 @@ bedrock.events.on('bedrock.started', async () => {
   // Ensure we filter out the local peer from the list of peers
   const localPeerId = await ledgerNode.consensus._localPeers.getPeerId(
     {ledgerNodeId: ledgerNode.id});
-
+  // creates a peerUrl at the node's address
+  const _getPeerUrl = ({peerId, hostname}) => `https://${hostname}` +
+          `/consensus/continuity2017/peers/${encodeURIComponent(peerId)}`;
```

Comment:
There's an indentation problem here, but really, this should be a helper function that is moved into the helper function down below instead of being a lambda we create right here.

```suggestion
  const _getPeerUrl = ({peerId, hostname}) => `https://${hostname}` +
    `/consensus/continuity2017/peers/${encodeURIComponent(peerId)}`;
```